### PR TITLE
fix(offline/push): add missing entity key and unix_milli

### PR DESCRIPTION
**Pull request by wfxr**
Close #907

### CODE REVIEW

File: internal/database/offline/sqlutil/push.go [hunk 1]
```diff
@@ -10,7 +10,20 @@ import (
...
+	columns := make([]string, 0, len(pushOpt.FeatureNames)+2)
+	columns = append(columns, pushOpt.EntityName, "unix_milli")
+	columns = append(columns, pushOpt.FeatureNames...)
+
+	records := make([]interface{}, 0, len(pushOpt.Records))
+	for _, r := range pushOpt.Records {
+		record := make([]interface{}, 0, len(r.Values)+2)
+		record = append(record, r.EntityKey, r.UnixMilli)
+		record = append(record, r.Values...)
+		records = append(records, record)
```
Comment (jinghancc) [inline internal/database/offline/sqlutil/push.go:23]: use `record.ToRow()`?

File: pkg/oomstore/types/feature_values.go [hunk 2]
```diff
@@ -14,3 +14,15 @@ func (fv *FeatureValues) FeatureValueSlice() []interface{} {
...
+type StreamRecord struct {
+	EntityKey string
+	UnixMilli int64
+	Values    []interface{}
+}
+
+func (r *StreamRecord) ToRow() []interface{} {
+	row := []interface{}{r.EntityKey, r.UnixMilli}
+	row = append(row, r.Values...)
+	return row
```
[the remark this answers is not in the document]
Comment (wfxr) [reply pkg/oomstore/types/feature_values.go]: fixed

File: internal/database/offline/sqlutil/push.go (hunk 1 above)
Comment (wfxr) [reply internal/database/offline/sqlutil/push.go]: updated

**Review by jinghancc — APPROVED** [review]
LGTM

**Pull request merged**